index.html

```diff
@@ -20,9 +21,10 @@
   <main>
     <div> <img src="images/smiling dog.jpg" alt="a puppy that is smiling"</div>
     <div>
-      <h1>Section Title</h1>
+      <h2>About Victoria</h2>
+      <p>Information about Victoria</p>
     </div>
   </main>
 </body>
-<footer></footer>
+<footer> (c) 2019 Victoria Herow</footer>
```

Comment:
You can use `&copy;` to get the copyright symbol.

Reply:
all set. thanks!